playbooks/gce/openshift-node/terminate.yml

```diff
@@ -16,7 +16,7 @@
   hosts: localhost
   connection: local
   tasks:
-    - name: Terminate minion instances
+    - name: Terminate node instances
```

Comment:
+1, however how long do we expect the hosts to live? Once they outlive their usefulness, we should finish killing off the directories.